web-frontend/src/App.js

```diff
@@ -96,6 +12,19 @@ import ForgotPassword from './routes/ForgotPassword/ForgotPassword';
 
 
 function App() {
+
+  useEffect(() => {
+    (function (d, m) {
+      var kommunicateSettings =
+        { "appId": "6ba050337f189922ca5b3c0e644a05e", "popupWidget": true, "automaticChatOpenOnNavigation": true };
```

Comment:
Credentials need to be stored in separate configuration files, not in the source code.